static/docs/user-guide/dvc-files-and-directories.md

```diff
@@ -40,10 +41,10 @@ Once initialized in a project, DVC populates its installation directory
 
 ## Structure of cache directory
 
-The structure of directory depends if the data is stored in a file or is in a
-directory.
+There are two ways in which the data is stored in cache which depends on if the
+actual data is stored in a file (eg. `data.csv`) or it is a directory of files.
 
-In case of a file, it is converted to a MD5 checksum which is a 32 characters
+The data file is converted to a checksum, usually MD5, which is a 32 characters
```

Comment:
file is not converted to a checksum.

we calculate a checksum and then we use it as a file name inside the cache